polymorphic_test.ts

```diff
@@ -210,13 +206,13 @@ test({
     class TestClass extends AbstractClass {
       @SerializeProperty("class")
       @PolymorphicSwitch(() => new TestClass(), "TestClass")
-      public [symbol]: string;
+      public [symbol]!: string;
```

Comment:
Is this a `definite assignment` out in the wild? [typescript doc](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#definite-assignment-assertions)

Reply:
if you don't have `!` then tsc expects a value at definition or in the constructor, this class did neither (and doesn't seen to set it) so we can use `!` to let tsc know it'll be set when we use it.

Reply:
FYI this is super helpful with `Maps`

```ts
const map = new Map<number, number>();
map.set(1,1);
const value = map.get(1)!; //this `!` means it'll is guaranteed to be set
```